tools/static-assets/skel-tailwind/tests/main.js

```diff
@@ -0,0 +1,20 @@
+import assert from "assert";
+
```

Comment:
I think it'd be good to create a few tests that check if Tailwind is configured and working correctly. Maybe checking if the required files exist? And/or if the rendered app has Tailwind classes applied?

Reply:
As we found out, skeletons are automatically tested [here](https://github.com/meteor/meteor/blob/c03ab0f4262e9a05c14aba3e6cbdee2f88477e30/tools/tests/create.js#L50)